### PR TITLE
Added Warning for Mac Mini - Ubuntu users

### PR DESCRIPTION
**Pull request by AdityaGarg8**

### CODE REVIEW

**Comment by AdityaGarg8** [comment]
This pr needs your review @Redecorating. The only case of Mac mini I have faced was successful in booting Ubuntu using refind only. You may check the discord in mbp-ubuntu

File: docs/index.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ A list of those devices can be found [on Apple's website](https://support.apple.
 
 - Some kernel versions, notably `5.8.x` and `5.9.x` are known to have issues. See the releases (those not marked as a pre-release) of [aunali1/linux-mbp-arch](https://github.com/aunali1/linux-mbp-arch/releases) for ones that have been tested. Keep in mind that some might very have reached their [end of life](https://en.wikipedia.org/wiki/Linux_kernel_version_history#Releases_5.x.y).
 - Newer versions of `iwd` currently have issues with non Big Sur patched kernels due to certain [p2p related checks missing](https://github.com/aunali1/linux-mbp-arch/blob/master/9001-brcmfmac-p2p-Ensure-virtual-interface-is-initialized.patch). If you are experiencing connection issues after following the [wifi guide](https://wiki.t2linux.org/guides/wifi/) you might want to downgrade to version `1.13` and hold updates for the package.
+- **Ubuntu** is not booting using the Mac Startup Manager on **Mac Mini**. Thus Mac Mini users are advised to [install the rEFInd bootloader](https://wiki.t2linux.org/guides/refind/) and boot Ubuntu using it instead of the Startup Manager.
```
Comment (Redecorating) [inline docs/index.md:12]: Can you add that it's when grub is used, i.e.:
> ...Manager on Mac Mini, when the default bootloader (GRUB) is used. Thus..

Can you also mention that it isn't known why this occurs.

File: docs/distributions/ubuntu/installation.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Download the latest safe release
 
-Warning: Versions newer than 5.7.19 are considered in-development builds and **should not** be used due to issues from within the kernel and the init system.
+Warning: Mac Mini users will need to [install rEFInd](https://wiki.t2linux.org/guides/refind/) as the Mac Startup Manager on Mac Mini is not booting Ubuntu.
```
Comment (Redecorating) [inline docs/distributions/ubuntu/installation.md:3]: Probably add ", with the default bootloader (GRUB)." to the end of this.
Also can you mention that it is unknown why this is happening.

**Review by Redecorating — CHANGES_REQUESTED** [review]
Is the issue that ubuntu doesn't show up in startup manager?

**Comment by AdityaGarg8** [comment]
> Is the issue that ubuntu doesn't show up in startup manager?

Ubuntu shows up. But on booting it gets stuck at a blank screen and force shuts down. Also, refind is used here just to boot grub. So it's like refind-grub-ubuntu

**Comment by Redecorating** [comment]
might have something to do with the macmini having no internal display, but that's *very* strange

**Comment by AdityaGarg8** [comment]
> might have something to do with the macmini having no internal display, but that's _very_ strange

Indeed strange. I have clarified this.

**Review by Redecorating — APPROVED** [review]

**Pull request merged**